frontend/web/src/features/hostdashboard/hostcalendar/HostCalendar.scss

```
@@ -0,0 +1,416 @@
+/* Root palette + layout */
+$bg: #fff;
+$card: #f5f5f5;
+$muted: #22181c;
+$text: #22181c;
+$line: #2a3140;
+
+$blue: #4e7cff; // booked
+$green: #22c55e; // available
+$red: #ef4444; // blocked
+$amber: #f59e0b; // maintenance
+$cyan: #06b6d4; // seasonal price
```

if these colours are only for specific cases, why not put that in the name instead of an ambiguous 'blue' or 'green'?

```suggestion
$booked-color: #4e7cff;
$available-color: #22c55e;
$blocked-color: #ef4444;
$maintenance-color: #f59e0b;
$seasonal-price-color: #06b6d4;
```